src/frontend/typer/barn.ts

```diff
@@ -60,6 +60,7 @@ export enum barnDataKeySpørsmål {
     søkersSlektsforhold = 'søkersSlektsforhold',
     søkersSlektsforholdSpesifisering = 'søkersSlektsforholdSpesifisering',
     borMedAndreForelder = 'borMedAndreForelder',
+    barnetsAdresse = 'barnetsAdresse',
```

Comment:
Egentlig ble jeg litt usikker på navngivning her. Den ligger jo under barn så jeg synes det holder med å bare hete adresse i såfall. Så det samme med omsorgsperson egentlig, at når den ligger under omsorgsperson så kanskje det feks bare burde hete navn, i stedet for omsorgspersonNavn. Vi må jo inn å endre noen ting i kontrakten enda, så er fort å gjort å endre i såfall. Kan ta en call på det eventuelt 😊

Reply:
Enig 😊 Jeg skal ta å endre ting i api og mottak etterpå, så kan jo gjøre de endringene samtidig 😊

Reply:
Startet å se litt på omsorgsperson omskrivning nå, og lurer litt på om jeg burde beholde spørsmålsiden eller ikke, er kanskje litt misvisende om den heter navn og ligger rett på EøsBarnSpørsmålID ? 
![image](https://user-images.githubusercontent.com/11887409/159919856-976f79c7-90f9-43f5-9611-7f55368869bb.png)

Reply:
Ja, jeg tenker denne endringen kun på typen til barn og omsorgsperson 😊 ikke felter i skjemaet eller noen av de andre tingene.